bg_atlasgen/atlas_scripts/princeton_mouse.py

```diff
@@ -0,0 +1,232 @@
+__version__ = "0.1"
+__atlas__ = "princeton_mouse"
+
+import tifffile
+import os.path
+import numpy as np
+import pandas as pd
+import json
+import time
+import multiprocessing as mp
+
+from rich.progress import track
+from pathlib import Path
+from bg_atlasapi import utils
+from scipy.ndimage import zoom
+
+from bg_atlasgen.mesh_utils import create_region_mesh, Region
+from bg_atlasgen.wrapup import wrapup_atlas_from_data
+from bg_atlasapi.structure_tree_util import get_structures_tree
+
+PARALLEL = False
+
+def create_atlas(working_dir, resolution):
+    # Specify information about the atlas:
+    ATLAS_NAME = __atlas__
+    SPECIES = "Mus musculus"
+    ATLAS_LINK = "https://brainmaps.princeton.edu/2020/09/princeton-mouse-brain-atlas-links/"
+    CITATION = "Pisano et al 2021, https://doi.org/10.1016/j.celrep.2021.109721"
+    ORIENTATION = "asl"
```

Comment:
I think this should be `las` (or `ras`), it's the orientation of the "raw" atlas files, which will then be transformed to asr upon packaging.